app/src/features/FhirResourceTree/FhirResourceTree.tsx

```diff
@@ -95,6 +99,28 @@ const FhirResourceTree = (): JSX.Element => {
     }
   }, [expandedNodes, hasAlreadyExpandedToTarget, nodeAncestorsIds]);
 
+  const createStaticInputWithFixedValue = (
+    node: ElementNode,
+    inputGroup: InputGroup
+  ) => {
+    const fixedValueKey = Object.keys(
+      node.definitionNode.definition
+    ).find((elementDefinitionKey) => elementDefinitionKey.includes("fixed"));
```

Comment:
More over, you're searching on a key to get a value, then it would be also more appropriate to work on `Object.entries` than `Object.keys`, like so :

```suggestion
   const fixedEntry = Object.entries(
      node.definitionNode.definition
    ).find(([key]) => key.startsWith("fixed"));
```